pkg/azurefile/utils.go

```diff
@@ -265,3 +275,47 @@ func replaceWithMap(str string, m map[string]string) string {
 	}
 	return str
 }
+
+func getAzcopyJob(dstFileshare string) (AzcopyJobState, string, error) {
```

Comment:
add comments for this function, and show what's the output for `azcopy jobs list | grep %s -B 3`

Reply:
done